5calls/app/src/main/res/layout/item_outcome.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Button xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/outcome_button"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:minHeight="@dimen/accessibility_min_size"
```

Comment:
Could you make the buttons the same look as before? I like the square corners and taller height: add these two lines:

android:background="@color/button_color"
android:layout_marginTop="8dp"

Reply:
Changed the background back to what it was before. Can't use `marginTop` for RecyclerViews. I am using an `ItemDecoration` in order to have the spacing.